quokka/core/fields.py

```diff
@@ -40,7 +40,8 @@ def inner(*args, **kwargs):
 def _get(self):
     def inner(*args, **kwargs):
         values = only_matches(self, kwargs)
-        if len(values) > 1:
+        values = list(values)
+        if len(list(values)) > 1:
```

Comment:
We should refactor this piece...I didn't see that because I did keep my focus on python3 support.
I'll fix this flags. =]